dashboard/views_test.py

```diff
@@ -38,6 +39,12 @@ def setUpTestData(cls):
             uid="{}_edx".format(cls.user.username),
             extra_data={"access_token": "fooooootoken"}
         )
+        cls.user_access_cache = UserCacheRefreshTimeFactory(
```

Comment:
It doesn't look like we use this anywhere. Does it have some other effect on the tests?